### ISSUE TITLE
Crash when I use node-sass module

### ISSUE THREAD

**Issue by yurist38** [Izhaki/nodemon-webpack-plugin#79]
Hello! I'm facing this issue right now: when I import `node-sass` in my code webpack starts crashing with this error message:

```
webpack:///./node_modules/node-sass/lib/binding.js?:15
      throw new Error(errors.missingBinary());
      ^

Error: Missing binding /Users/yuridrabik/projects/cvmaker-templates/vendor/darwin-x64-64/binding.node
Node Sass could not find a binding for your current environment: OS X 64-bit with Node.js 10.x

Found bindings for the following environments:

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass` to download the binding for your current environment.
    at module.exports (webpack:///./node_modules/node-sass/lib/binding.js?:15:13)
    at eval (webpack:///./node_modules/node-sass/lib/index.js?:14:94)
    at Object../node_modules/node-sass/lib/index.js (/Users/yuridrabik/projects/cvmaker-templates/dist/index.js:2698:1)
    at __webpack_require__ (/Users/yuridrabik/projects/cvmaker-templates/dist/index.js:20:30)
    at eval (webpack:///./src/HtmlBuilder/index.ts?:9:21)
    at Object../src/HtmlBuilder/index.ts (/Users/yuridrabik/projects/cvmaker-templates/dist/index.js:3691:1)
    at __webpack_require__ (/Users/yuridrabik/projects/cvmaker-templates/dist/index.js:20:30)
    at eval (webpack:///./src/index.ts?:9:39)
    at Object../src/index.ts (/Users/yuridrabik/projects/cvmaker-templates/dist/index.js:3703:1)
    at __webpack_require__ (/Users/yuridrabik/projects/cvmaker-templates/dist/index.js:20:30)
[nodemon] app crashed - waiting for file changes before starting...
```

Of course, I tried to reinstall and rebuild node-sass (with `--force` flag too). It doesn't fix the problem. Globally installed `node-sass` works fine, without `nodemon-webpack-plugin` the project is working.

OS: Last Mac OS X Catalina, 64 bit
Node: 10.16.0
NPM: 6.12.0

Do you know what can be a problem? Thanks!

**Comment by Izhaki** [comment]
Doesn't seem to be plugin-related.

Are you sure the build passes without `nodemon-webpack-plugin`? Which version of the plugin do you use?

I can only troubleshoot it if you share a reproducible example.

**Comment by yurist38** [comment]
@Izhaki Thanks for replying! I've actually managed to fix this particular issue by adding `node: { __dirname: true }` to my webpack config. But then I've got another issue and it's not related to the plugin, I'm pretty sure. So I think this issue can be closed. Thanks again!
